librz/cons/dietline.c

```diff
@@ -431,6 +438,206 @@ RZ_API int rz_line_hist_chop(const char *file, int limit) {
 	return 0;
 }
 
+static inline bool is_valid_undo_entry(const RzLineUndoEntry *e) {
+	if (!e) {
+		return false;
+	}
+	if (e->start < 0 || e->start > e->end) {
+		return false;
+	}
+	if (e->type == UNDO_DELETE || e->type == UNDO_INSERT) {
+		if (RZ_STR_ISEMPTY(e->text)) {
+			return false;
+		}
+	}
+	return true;
+}
+
+static inline bool __nothing_to_do(bool is_redo) {
```

Comment:
avoid names starting with `__`

Reply:
also if you are creating static functions, avoid to set the `inline` attribute..